python_modules/dagster/dagster/_core/storage/pipeline_run.py

```diff
@@ -645,23 +646,25 @@ def __new__(
         )
 
     @property
-    def pipeline_name(self):
-        return self.job_name
+    def pipeline_name(self) -> str:
+        return self.job_name  # type: ignore  # (possible none)
```

Comment:
this should be typed as optional - filtering on job name is optional

Reply:
good catch